### PR TITLE
Homework2

### PR DESCRIPTION
**Pull request by adrianmasiewicz**

### CODE REVIEW

File: module2/homework/addEven/addEven.hpp [hunk 1]
```diff
@@ -0,0 +1,18 @@
+#pragma once
+#include <vector>
+
+int addEven(const std::vector<int>& numbers) {
+    int liczba; //liczba elementów w wektorze
+    int liczba3 = 0; //suma liczb zwracana przez funkcję
```
Comment (DamiXioR) [inline module2/homework/addEven/addEven.hpp:6]: please remove comments

Comment (DamiXioR) [inline module2/homework/addEven/addEven.hpp:6]: remove comments

File: module2/homework/maxOfVector/maxOfVector.hpp [hunk 2]
```diff
@@ -0,0 +1,20 @@
+#pragma once
+#include <vector>
+#include <limits>
+
+int maxOfVector(const std::vector<int> numbers) {
+    int liczba; //liczba elementów w wektorze
+    int liczba3; //największa wartość zwracana przez funkcję
```
Comment (DamiXioR) [inline module2/homework/maxOfVector/maxOfVector.hpp:7]: remove comments

File: module2/homework/nwdNww/nwdNww.hpp [hunk 3]
```diff
@@ -0,0 +1,26 @@
+#pragma once
+
+int NWD(int lhs, int rhs) { 
+
+    if (lhs < 0) lhs *= -1;
```
Comment (DamiXioR) [inline module2/homework/nwdNww/nwdNww.hpp:5]: please use curly braces with if even You have one line code

**Comment by ziobron** [comment]
I can see that there is #225 which is raised against a correct branch. Let me close this one.

**Pull request closed without merging**